deploy/grafana/dashboards/pod-level.json

```diff
@@ -1062,25 +1062,39 @@
   ],
   "templating": {
     "list": [
+      {
+        "current": {
+          "text": "Prometheus",
+          "value": "Prometheus"
+        },
+        "hide": 0,
+        "label": null,
+        "name": "DS_PROMETHEUS",
+        "options": [],
+        "query": "prometheus",
+        "refresh": 1,
+        "regex": "",
+        "type": "datasource"
+      },
       {
         "current": {},
         "datasource": {
           "type": "prometheus",
-          "uid": "${datasource}"
+          "uid": "${DS_PROMETHEUS}"
         },
-        "definition": "query_result(retina_adv_forward_count)",
+        "definition": "query_result(networkobservability_adv_forward_count)",
         "hide": 1,
         "includeAll": true,
-        "label": "Pod Name ",
+        "label": "Pod Name",
         "multi": false,
         "name": "pod",
         "options": [],
         "query": {
-          "query": "query_result(retina_adv_forward_count)",
+          "query": "query_result(networkobservability_adv_forward_count)",
           "refId": "StandardVariableQuery"
         },
         "refresh": 1,
-        "regex": "/.*_podname=\"([^\"]*).*/",
+        "regex": "/.*podname=\"([^\"]*).*/",
```

Comment:
nice catch. You must have been using the advanced local-context metric mode. Just noting how this prompted initial thoughts on #344